packages/nodes-base/nodes/Onfleet/descriptions/TaskDescription.ts

```diff
@@ -0,0 +1,431 @@
+import {
+	INodeProperties
+} from 'n8n-workflow';
+
+export const taskOperations = [
+	{
+		displayName: 'Operation',
+		name: 'operation',
+		type: 'options',
+		displayOptions: {
+			show: {
+				resource: [
+					'tasks',
+				],
+			},
+		},
+		options: [
+			{
+				name: 'Add',
+				value: 'create',
+				description: 'Add new Onfleet task.',
+			},
+			{
+				name: 'Add multiple tasks',
```

Comment:
This is also a bit confusing when using `add`

Reply:
I would think `Creating multiple tasks in batch` sounds better

Reply:
Description changed